test/integration/httpd_role_test.lua

```diff
@@ -146,3 +171,32 @@ g.test_change_server_addr_on_the_run = function(cg)
     t.assert_equals(resp.status, 200, 'response not 200')
     t.assert_equals(resp.body, 'pong')
 end
+
+g.test_log_requests = function(cg)
+    t.skip_if(cg.params.use_tls)
+
+    local function make_request(port)
+        local resp = http_client:get(string.format('http://localhost:%d/ping', port))
+        t.assert_equals(resp.status, 200, 'response not 200')
+    end
+
+    local function assert_should_log(expected)
+        local grep_res = cg.server:grep_log('GET /ping', math.huge)
+        if expected then
+            t.assert(grep_res)
+        else
+            t.assert_not(grep_res)
+        end
+    end
+
+    local log_level = tonumber(cg.params.log_level)
+
+    make_request(13002)
```

Comment:
I think it's better to have something like
```suggestion
    make_request('localhost:13002')
```
or
```suggestion
    make_request({port = 13002})
```
otherwise five digit number is confusing.